internal/trafficlogs/logger.go

```diff
@@ -90,35 +106,70 @@ func (l *FileBasedLogger) logRequest(id string, request *http.Request, body stri
 	if err != nil {
 		return err
 	}
-	_, err = l.requestLogFile.WriteString(fmt.Sprintf("Request-ID: %s\n%s\n%s\n=========================\n", id, string(dump), body))
+
+	// write id
+	_, err = l.requestLogFile.WriteString(fmt.Sprintf("Request-ID: %s\n", id))
 	if err != nil {
 		return err
 	}
 
+	// write dump
+	if _, err := l.requestLogFile.WriteString(fmt.Sprintf("%s", string(dump))); err != nil {
```

Comment:
you've lost the newline after the dump I think

Reply:
for some reason there are alreade two newlines in the dump data itself :D